src/Microsoft.Health.Dicom.Core/Features/ExtendedQueryTag/AddExtendedQueryTagService.cs

```diff
@@ -17,7 +18,9 @@ namespace Microsoft.Health.Dicom.Core.Features.ExtendedQueryTag
 {
     public class AddExtendedQueryTagService : IAddExtendedQueryTagService
     {
-        private readonly IExtendedQueryTagStore _extendedQueryTagStore;
+        // Different from Asp.net Core, we couldn't get schemaVersion before AzureFunction execution.
```

Comment:
I don't understand this comment. Maybe AzureFunction needs to fixed on when it gets initialized

Reply:
In DicomServer scenario, we get schema version through [SchemaJobWorkerBackgroundService](https://github.com/microsoft/healthcare-shared-components/blob/master/src/Microsoft.Health.SqlServer.Api/Schema/SchemaJobWorkerBackgroundService.cs) before other Dicom components (e.g AddExtendedQueryTagService), so that the components can initialize correctly.
But I didn't find similar things in Azure Function -- [HostedService is not part of Azure Function Today](https://github.com/Azure/azure-functions-host/issues/5447#issuecomment-575368316).

Reply:
Not true, we need to initialize the SqlServer components before initializing the backgroundworkers.
You just need to do the same in Azure functions. The order in how you initialize things matter